Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml

```diff
@@ -66,6 +94,10 @@
   bodyPart: RLeg
   markingCategory: Legs
   speciesRestriction: [Human]
+  coloring:
+    default:
+      !!type:TattooColoringColor
```

Comment:
i believe this is causing your tests to fail:

```
  ----> System.AggregateException : One or more errors occurred. (One or more errors occurred. (One or more errors occurred. (CLIENT: 5.435s [ERRO] proto: Reading Content.Shared.Humanoid.Markings.MarkingPrototype(TattooCampbellRightLeg) threw the following exception: System.InvalidOperationException: Instances of abstract classes cannot be created.
```

Reply:
yes I fixed it in my local branch